source/frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -114,13 +137,22 @@ export const pointToLayer = (feature: ICluster, latlng: LatLngExpression): Layer
  * Get an icon type for the specified cluster property details.
  */
 export const getMarkerIcon = (feature: ICluster, selected?: boolean) => {
+  if (feature.properties.IS_PAYABLE_LEASE || feature.properties.isPayableLease) {
+    if (selected) {
+      return propertyWithLeaseIconSelect;
+    } else {
+      return propertyWithLeaseIcon;
+    }
+  }
+
```

Comment:
Did you update the marker legend to include this new pin? I mean this map component...
![image](https://user-images.githubusercontent.com/24322224/207740679-c57d3fe7-f7e1-4d7b-a0ce-9b0a58b33ef7.png)

Reply:
uh nope. Never pressed that button before. Guess thats what they meant by 'label' :)